esp/public/media/scripts/program/modules/scheduling_checks.jsx

```diff
@@ -28,8 +37,29 @@ var SchedulingCheck = React.createClass({
       open: false,
       failed: false,
       timestamp: "never",
+      tableState: {
+        greyed: {},
+        sort: -1,
+        reverse: false
+      }
     };
   },
+  
+  updateTableState: function (state) {
+    this.state.tableState = state;
+  },
+  
+  resetTable: function () {
+    this.setState({
+      tableState: {
+        greyed: {},
+        sort: -1,
+        reverse: false
+      },
+      open: false
```

Comment:
I could possibly try to implement it so that it's unnecessary, but currently I have it so that rendering the child updates the parent's record of the child's state. Reset is supposed to force the child to be recreated, but if I don't close the box the child's state will "update" the parent's record and then we will be sad.